charts/kubernetes/templates/topology.yaml

```diff
@@ -1,3 +1,16 @@
+---
+apiVersion: mission-control.flanksource.com/v1
```

Comment:
I think only create if prometheus url is specified and `prometheus.createConnection` is true (i.e. we might want to use an existing connection, or share the same connection amongst multiple topologies